swe.py

```diff
@@ -536,6 +537,14 @@ def init_file(name, cnfg, save):
         required=False, 
         default=False, help="Disable coriolis terms.")
 
+    parser.add_argument(
+        "--betas", dest="betas", type=float,
```

Comment:
I wonder if there's a way that we could do this in a general way that might extend to other FB integration schemes in the future? [This post](https://stackoverflow.com/questions/20165843/argparse-how-to-handle-variable-number-of-arguments-nargs) suggests that it's possible to setup arbitrary length array inputs in the argparser, so we could use this for `RK22-FB` as well, for example. The question then is just how to handle the defaults --- what about if this array is default empty in the argparser, and only overrides the hard-coded values in the `step_RKXX` routines when it's explicitly user-defined?
How about also a slight name change to `--FB-weight`, just to make it a bit more descriptive?